File: Upcoder/Other/Tính tổng giá trị âm trong Mảng/LTCB-126.cpp
#include <bits/stdc++.h>
using namespace std;

float SumAm(float a[], int n) 
{
    float Tong = 0.0;
    for (int i = 0; i < n; i++) {
        cin >> a[i];
        if (a[i] < 0)
            Tong += a[i];
    }
    return Tong;
}

int main() {
    int n;
    cin >> n;
    if (n <= 0) {
        cout << -1;
        return 0;
    }
    float a[10];
    float tong = SumAm(a, n);
     cout <<(round(tong * 100))/100;
    return 0;
}

File: Upcoder/Other/Tính tổng giá trị âm trong Mảng/LTCB-126.cs
//Code này 8.68
using System;
using System.Linq;

namespace Tong 
{
    public class Program
    {
        static float SumAm(float[] a, int n)
        {
            float Tong = 0.0f;
            for (int i = 0; i < n; i++)
            {
                if (a[i] < 0)
                    Tong += a[i];
            }
            return Tong;
        }

        static void Main(string[] args)
        {
            var input = Console.ReadLine().Split(' ');

            int n = int.Parse(input[0]);
            if (n <= 0)
            {
                Console.Write("-1");
                return;
            }

            float[] a = input.Skip(1).Take(n).Select(float.Parse).ToArray();

            float tong = SumAm(a, n);
            Console.Write(Math.Round(tong, 2));
        }
    }
}
